com.typesafe.hocon/src/com/typesafe/config/Hocon.xtext

```diff
@@ -18,7 +18,7 @@ Root:
 
 Object:
  '{' NL*
-    ( members+=Member (',' | NL) NL* )*  
+    ( members+=Member ','? NL* )*  
```

Comment:
Probably you are right. Actually last week I tried the same thing as in `Root` but I could not get it working. So I ended up with this solution which, as you said, probably works by accident. I won't try anymore, as I am under time pressure the next weeks.